### DISCUSSION TITLE
METViewer Test data on git

### DISCUSSION THREAD

**Discussion by rgashrit** [dtcenter/METplus#1164] [Plot Generation] [answered]
Hi..
The METviewer test data is no longer available at the following link

https://github.com/dtcenter/METViewer-test.git

Please point to changed/correct location

**Comment by georgemccabe** [comment]
Hi @rgashrit,

It looks like this repository is currently a private repository, so it is unavailable to the public. @TatianaBurek is the owner of the repository, but she is out on vacation for the next two weeks. She can provide more information when she returns. How are you using these data? I am not very familiar with the files but they appear to be used for internal testing of the METviewer software.

**Comment by TatianaBurek** [comment, marked as answer]
Hi @rgashrit,
You can find MET output sample data here:
https://dtcenter.ucar.edu/dfiles/code/METplus/METviewer/sample_data-met_out_v9.1.tgz
